pkg/minikube/cluster/cluster.go

```diff
@@ -618,3 +618,17 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 
 	return serviceURLs, nil
 }
+
+//Create the host directories in the virtual machine if not present
+func EnsureHostDirectories(api libmachine.API, dirs []string) (error) {
```

Comment:
I think minishift/pkg/minishift/util/oc_cluster.go is the right location for this function.